banking/klarna_kosma_integration/doctype/bank_reconciliation_tool_beta/bank_reconciliation_tool_beta.py

```diff
@@ -143,14 +145,20 @@ def create_journal_entry_bts(
 				"cost_center": get_default_cost_center(company),
 			},
 			{
-				"account": company_account,
+				"account": company_bank_account,
 				"bank_account": bank_transaction.bank_account,
 				"credit_in_account_currency": bank_credit_amount,
 				"debit_in_account_currency": bank_debit_amount,
 				"cost_center": get_default_cost_center(company),
 			},
 		],
 	)
+
+	company_currency = get_company_currency(company)
+	journal_entry.multi_currency = any(
+		currency != company_currency for currency in [bank_account_currency, second_account_currency]
+	)
```

Comment:
```suggestion
	journal_entry.multi_currency = (
		0 if company_currency == bank_account_currency == second_account_currency else 1
	)
```

Bit easier for me to understand